src/coding-guidelines/expressions.rst

```diff
@@ -783,15 +782,17 @@ Expressions
 
 .. guideline:: Integer shift shall only be performed through `checked_` APIs
     :id: gui_RHvQj8BHlz9b 
-    :category: required
+    :category: advisory
     :status: draft
     :release: 1.7.0-latest
     :fls: fls_sru4wi5jomoe
     :decidability: decidable
     :scope: module
     :tags: numerics, reduce-human-error, maintainability, portability, surprising-behavior, subset
 
-    In particular, the user should only perform left shifts via the `checked_shl <https://doc.rust-lang.org/core/index.html?search=%22checked_shl%22>`_ function and right shifts via the `checked_shr <https://doc.rust-lang.org/core/index.html?search=%22checked_shr%22>`_ function. Both of these functions exist in `core <https://doc.rust-lang.org/core/index.html>`_.
+    In particular, the user should only perform left shifts via the `checked_shl <https://doc.rust-lang.org/core/index.html?search=%22checked_shl%22>`_
+    function and right shifts via the `checked_shr <https://doc.rust-lang.org/core/index.html?search=%22checked_shr%22>`_ function.
+    Both of these functions exist in `core <https://doc.rust-lang.org/core/index.html>`_.
```

Comment:
@rcseacord you added these newlines to account for this bug, right? https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/265